api/client/samples/crop_models/brazil_soybeans.ipynb

```diff
@@ -0,0 +1,1581 @@
+{
```

Comment:
<p>Do we want to leave it at that or back this statement up with a rudimentary regression and give an r^2. </p>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
yup, sorry forgot to add teh TODO, but  I've added 2 simple regression models and their in-sample testing stats